learnbot-dsl/LearnBotClient.py

```diff
@@ -3,7 +3,7 @@
 
 import sys, traceback, Ice, os, math, time, json, ast, copy
 import json
-import cv
+# import cv
```

Comment:
Because `cv2` is being used instead of `cv`, so instead of removing this line, I just commented it out, in case it is required later.